omnibus/Gemfile.lock

```diff
@@ -1,19 +1,22 @@
 GIT
   remote: git://github.com/chef/omnibus-software.git
-  revision: 40fbb9d412a2ebc17c06ff34922682de28970127
+  revision: b1a068c75d4eb0833c6c3884f7e0692ab1f29fb1
+  branch: ksubrama/ruby23
```

Comment:
I'm really not thrilled with the number of projects where we are now on this branch.  We should help get it merged if possible.